Assets/Scripts/Player/PlayerController.cs

```diff
@@ -119,6 +119,20 @@ public class PlayerController : MonoBehaviour {
 	public float AirJumpLength => airJumpEndTime;
 	public float SpeedMultiplier { set => speedMultiplier = value; }
 
+#if UNITY_EDITOR
```

Comment:
Did you forget to add development build here?

Reply:
Yup, good catch!